Sources/HipMobileUI.Droid/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is this file needed for?

Reply:
I don't know. This file was automaticly created. I will remove this one and the other one (iOS/app.config)